code/src/Components/MovieCard/movieCard.css

```diff
@@ -0,0 +1,77 @@
+.movieCard-container{
+    background: white;
+    position: relative;
+    color: #fff;
+    text-decoration: none;
+    margin: 0px 2px;
+    background:darkblue;
+}
+
+.movieCard-overlay{
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    background: rgba(0,0,0,.75);
+    display: none;
+    padding: 20px;
+    z-index: 3;
+}
+
+.movieCard-image{
```

Comment:
When reviewing we saw that the second movie from the rights img on the first page is smaller than others/the blue background is not in line with other movies. So you could maybe set a fixed size to the pictures or adjust the height.